.travis.yml

```diff
@@ -1,66 +1,52 @@
+---
```

Comment:
We've just removed Travis CI support in #56. So this file does not exist anymore.

Reply:
Ok, fine, I will remove this file